docs/sources/datasources/loki/_index.md

```diff
@@ -15,81 +15,36 @@ weight: 800
 
 # Loki data source
 
-Grafana ships with built-in support for [Loki](/docs/loki/latest/), an open-source log aggregation system by Grafana Labs.
-This topic explains configuring and querying specific to the Loki data source.
+Grafana Loki is a set of components that can be combined into a fully featured logging stack.
+Unlike other logging systems, Loki is built around the idea of only indexing metadata about your logs: labels (just like Prometheus labels). Log data itself is then compressed and stored in chunks in object stores such as S3 or GCS, or even locally on a filesystem.
+
+The following guides will help you get started with Loki:
+
+- [Getting started with Loki](/docs/loki/latest/getting-started/)
+- [Install Loki](/docs/loki/latest/installation/)
+- [Loki best practices](/docs/loki/latest/best-practices/#best-practices)
+- [Configure the Loki data source](/docs/grafana/latest/datasources/loki/configure-loki-data-source/)
+- [LogQL](/docs/loki/latest/logql/)
+- [Loki query editor](/docs/grafana/latest/datasources/loki/query-editor/)
```

Comment:
I think here is a good opportunity to use a `relref`.
Hugo will make sure that the `query-editor` page exists when we build the docs in CI, or we run `make docs`, and will error if that page does not exist.

```suggestion
- [Loki query editor]({{< relref "./query-editor" >}})
```

Why not use `/docs/reference`?
Since this page is part of your data sources documentation, the relative path works for both this page at `/docs/grafana/latest/datasources/loki/` where it will link to `/docs/grafana/latest/datasources/loki/query-editor/` and also the Grafana Cloud page `/docs/grafana-cloud/connect-externally-hosted/data-sources/loki/` where it will link to `/docs/grafana-cloud/connect-externally-hosted/data-sources/loki/query-editor/`